include/kframework/evm-types.md

```diff
@@ -164,18 +164,20 @@ Bitwise logical operators are lifted from the integer versions.
                  | Int "|Word"   Int [function, total]
                  | Int "&Word"   Int [function, total]
                  | Int "xorWord" Int [function, total]
-                 | Int "<<Word"  Int [function]
-                 | Int ">>Word"  Int [function]
-                 | Int ">>sWord" Int [function]
- // -------------------------------------------
+                 | Int "<<Word"  Int [function, total]
+                 | Int ">>Word"  Int [function, total]
+                 | Int ">>sWord" Int [function, total]
+ // --------------------------------------------------
     rule ~Word W       => W xorInt maxUInt256
     rule W0 |Word   W1 => W0 |Int W1
     rule W0 &Word   W1 => W0 &Int W1
     rule W0 xorWord W1 => W0 xorInt W1
-    rule W0 <<Word  W1 => chop( W0 <<Int W1 ) requires W1 <Int 256
-    rule  _ <<Word  W1 => 0 requires W1 >=Int 256
-    rule W0 >>Word  W1 => W0 >>Int W1
-    rule W0 >>sWord W1 => chop( (abs(W0) *Int sgn(W0)) >>Int W1 )
+    rule W0 <<Word  W1 => chop( W0 <<Int W1 ) requires W1 <Int 256 andBool W1 >=Int 0
+    rule  _ <<Word  _  => 0 [owise]
+    rule W0 >>Word  W1 => W0 >>Int W1 requires W1 >=Int 0
+    rule  _ >>Word   _ => 0 [owise]
+    rule W0 >>sWord W1 => chop( (abs(W0) *Int sgn(W0)) >>Int W1 ) requires W1 >=Int 0
+    rule  _ >>sWord  _ => 0 [owise]
```

Comment:
```suggestion
    rule W0 <<Word  W1 => chop( W0 <<Int W1 ) requires 0 <=Int W1 andBool W1 <Int 256
    rule  _ <<Word  _  => 0 [owise]
    rule W0 >>Word  W1 => W0 >>Int W1 requires W1 >=Int 0
    rule  _ >>Word   _ => 0 [owise]
    rule W0 >>sWord W1 => chop( (abs(W0) *Int sgn(W0)) >>Int W1 ) requires 0 <=Int W1
    rule  _ >>sWord  _ => 0 [owise]
```

This is a stylistic choice, but one made in most places in this repo. Basically it lines up the numbers in the order they would appear on a number line. Also, we have lemmas turning the `>Int` and `>=Int` variants into `<Int` and `<=Int`, so we don't have to have as much duplication of lemmas. So just using the `<*Int` variants from the start removes one simplification rule that needs to apply to flip them.

Also, for the first one, we could use `#rangeUInt(8, W1)`, and it wolud mean the same thing.